src/app/dataset-block/metadata-block/components/event-details/components/base-event/base-event.component.ts

```diff
@@ -0,0 +1,56 @@
+import { MetadataEvent } from "./../../../../../../api/kamu.graphql.interface";
+import {
+    AfterViewChecked,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ComponentRef,
+    Input,
+    QueryList,
+    ViewChildren,
+    ViewContainerRef,
+} from "@angular/core";
+import { EventRow, EventSection } from "../../builder.events";
+import { BasePropertyComponent } from "../common/base-property/base-property.component";
+
+@Component({
+    selector: "app-base-event",
+    templateUrl: "./base-event.component.html",
+    styleUrls: ["./base-event.component.sass"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class BaseEventComponent implements AfterViewChecked {
```

Comment:
Since we don't support all of the possible events, maybe let's call it `BaseDynamicEventComponent`

Reply:
Renamed